src/utils/EscrowCalc.js

```diff
@@ -1,3 +1,6 @@
+const BN = require("bignumber.js");
+BN.config({ DECIMAL_PLACES: 18 });
```

Comment:
be careful with hardcoding this @manolingam. Need to use token-list or other to get the decimals for each token.

Reply:
Okay. Since it's _weth_ and _dai_, I left it with 18 decimals. But yeah, will keep that in mind if any other tokens are added in future.